utils/fill_zero_test.go

```diff
@@ -1,61 +1,77 @@
 package utils
 
 import (
+	"bytes"
 	"context"
+	"crypto/rand"
+	"fmt"
+	"io"
 	"os"
-	"strconv"
 	"testing"
 
+	"github.com/bmc-toolbox/common"
 	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/stretchr/testify/require"
 )
 
 func Test_NewFillZeroCmd(t *testing.T) {
-	// Test if NewFillZeroCmd returns a non-nil pointer
-	zw := NewFillZeroCmd(false)
-	if zw == nil {
-		t.Error("Expected non-nil pointer, got nil")
-	}
+	require.NotNil(t, NewFillZeroCmd(false))
 }
 
-func Test_WipeDisk(t *testing.T) {
-	for _, size := range []int{4095, 4096, 4097, 8192} {
-		t.Run(strconv.Itoa(size), func(t *testing.T) {
+func Test_FillZeroWipeDrive(t *testing.T) {
+	for _, size := range []int64{8192 - 1, 8192, 8192 + 1, 8192 * 2} {
```

Comment:
I'm not convinced bracketing a block-size boundary does all that much here.

Reply:
It did catch an issue in an earlier version of the code, but only when I ran stuff manually because this test has been buggy from the beginning :(. Its also cheap so not a big problem to keep around and keep the bug from returning.